.devilbox/www/include/lib/container/Httpd.php

```diff
@@ -60,10 +60,11 @@ public function getVirtualHosts()
 	{
 		$docRoot	= $this->_docRoot;
 		$vhosts		= array();
+		$htdocs		= loadClass('Helper')->getEnv('HTTPD_DOCROOT_DIR');
 
 		if ($handle = opendir($docRoot)) {
 			while (false !== ($directory = readdir($handle))) {
-				if ($this->_is_valid_dir($docRoot . DIRECTORY_SEPARATOR . $directory) && $directory != '.' && $directory != '..') {
+				if ($this->_is_valid_dir($docRoot . DIRECTORY_SEPARATOR . $directory. "/". $htdocs) && $directory != '.' && $directory != '..') {
```

Comment:
This will make a big difference and should be discussed with the community.

**Current behaviour:**

* Project exists, but does not have a vhost: Intranet will actually complain and let you know where to create the `htdocs` directory.

**New behaviour:**

* Project exists, but does not have a vhost: Intranet will not show this vhost.

---

The current behaviour is helping people a lot to get started with the Devilbox. The new one will counteract this, but will allow more advanced people to hide stuff they intentionally want to hide.